### PR TITLE
Readme Updates

### PR DESCRIPTION
**Pull request by poornima2796**
Minor updates to the `Contribution` section of the Readme doc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,7 +112,9 @@ Currently the project is being primarily co-ordinated via the GDG Melbourne slac
 
 ### Pull Request Process
 
+- `Fork` the project by clicking on the fork button.
```
Comment (lukesleeman) [inline README.md]: Might be good to split this out into sections for people who are part of the project team, and external contributors

**Review by domesticmouse — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (poornima2796) [reply README.md]: Just updated it again with the 2 sections. Lmk if any more changes are required!

File: README.md [hunk 2]
```diff
@@ -112,11 +112,19 @@ Currently the project is being primarily co-ordinated via the GDG Melbourne slac
...
 
+## Method 1 - If you are part of the team
 - Develop your work in a branch. Branch names should be prefixed with `feature`, `tech`, or `bugfix` depending on what is being done in them. eg: `feature\update-readme`
 - Open a pull request into master. Be sure to include a detailed description in your pull request. Other contributors will comment on your code or approve it
 - Please be sure to note the build status of your PR reported by the Codemagic CI
 - Once your pull request has been approved, please merge into master.
 
+## Method 2 - If you are not yet part of the team
+- If you are not a part of the team, you can still work on the code by clicking on the `Fork` option in this page.
+- Once you have forked it, you would be able to clone it and work on it.
```
Comment (lukesleeman) [inline README.md]:
```suggestion
- Once you have forked to your own github account,  you will be able to check out the fork and work on it.
```

File: README.md [hunk 3]
```diff
@@ -112,11 +112,19 @@ Currently the project is being primarily co-ordinated via the GDG Melbourne slac
...
+- To create a pull request, you would have to push your local changes back to your git account. 
+- You would then find the option to `Compare and Pull Request`. 
+- Please follow the instructions above on how to submit a pull request. 
+
```
Comment (lukesleeman) [inline README.md:128]: Can you put something here about approvals?  What actually happens?  Are you able to merge or does somebody on the team have to do it?

**Review by lukesleeman — CHANGES_REQUESTED** [review]
Its looking really good, great work 👍 

I've made a few minor suggestions.  After that you should be good to merge!

File: README.md (hunk 3 above)
Comment (poornima2796) [reply README.md:128]: I think someone on the team would have to merge it as I am not able to find the option to do it. Will update the readme with this!

**Review by lukesleeman — APPROVED** [review]
This looks good, I approve ✅ 😁

**Pull request merged**